products/views.py

```diff
@@ -1,3 +1,35 @@
-from django.shortcuts import render
+from django.urls import reverse_lazy
+from django.views.generic import ListView
+from .forms import ProductForm
+from .models import Product
+from common.mixins import TitleMixin, AddProductMixin, DeleteProductMixin
+from django.contrib.auth.mixins import LoginRequiredMixin
 
-# Create your views here.
+
+class ProductListView(TitleMixin, ListView):
+    title = 'Products'
+    model = Product
+    template_name = 'product_list.html'
+    context_object_name = 'products'
+
+    def get_context_data(self, **kwargs):
```

Comment:
You dont need rewrite `get_context_data` and specify varible for list product, because all this thing do django class ListView under the hood, just specify class attribute model

Detail - https://docs.djangoproject.com/en/5.0/ref/class-based-views/generic-display/#listview